### ISSUE TITLE
can i use my custom css framework like semantic ui or vuetify for native script

### ISSUE THREAD

**Issue by johnzeus14** [nativescript-vue/nativescript-vue#274]
can i use my custom css framework like semantic ui or vuetify for native script

**Comment by msaelices** [comment]
No, I don't think so. These CSS framework are web based and they are using the 100% of the browser capabilities, while the [NativeScript styling](https://docs.nativescript.org/ui/styling) is using only a subset of the CSS language.

**Comment by msaelices** [comment]
About the specific Vuetify case, the integration is even harder. Vuetify renders HTML DOM elements, but NativeScript-Vue uses the Vue virtual dom to render NativeScript components instead.
